.changeset/gentle-buttons-hunt.md

```diff
@@ -0,0 +1,22 @@
+---
+'@backstage/create-app': patch
+---
+
+Update the template to reflect the renaming of `CatalogResultListItem` to `CatalogResultListItemProps` from `@backstage/plugin-catalog`.
```

Comment:
```suggestion
Update the template to reflect the renaming of `CatalogResultListItem` to `CatalogSearchResultListItem` from `@backstage/plugin-catalog`.
```

Reply:
or do you mean `CatalogResultListItem` **and** `CatalogResultListItemProps`?

Reply:
Indeed :grin: copy-paste derp